documentation/src/main/asciidoc/stories/assembly_cache_modes.adoc

```diff
@@ -8,6 +8,25 @@
 * Optimizing databases by decreasing the volume of write operations.
 * Providing resiliency and durability for consistent data across clusters.
 
+[IMPORTANT]
+====
+{brandname} places a restriction of less than or equal to 255 characters on a cache name or a cache template name.
```

Comment:
I think there is a bit of fluff
places restriction > restricts/limits
less than equal > maximum